packages/react-priority-overflow/src/useOverflowMenu.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
+import { useOverflowContext } from './overflowContext';
+import { useOverflowCount } from './useOverflowCount';
+
+export function useOverflowMenu<TElement extends HTMLElement>(id = 'overflow-menu') {
```

Comment:
Is it a good idea to have a default value?

```ts
useOverflowMenu();
useOverflowMenu();
```

And it gets silently broken...

Reply:
yes, but now I can use `useId` :)

Thanks for pointing this out